### DISCUSSION TITLE
How to contribute to this project?

### DISCUSSION THREAD

**Discussion by rafaelrenanpacheco** [micronaut-projects/micronaut-security#497] [Q&A] [answered]
Hello!

I'm trying to make some contributions to this project, but I'm unable to run the same steps that works for micronaut-core. Here is what I have tried so far:

1. Clone the project
2. Run `./gradlew publishToMavenLocal`

Which results in the following error:

```
* What went wrong:
Execution failed for task ':security-annotations:generateMetadataFileForMavenPublication'.
> Invalid publication 'maven':
    - Publication only contains dependencies and/or constraints without a version. You need to add minimal version information
```

I also tried to link this project on a existing micronaut project, but the changes were just ignored.

Another try was to run `./gradlew security-jwt:publishToMavenLocal`, which didn't do nothing.

What is the best way to contribute and test the changes on an existent project?

**Comment by alvarosanchez** [comment, marked as answer]
The build was certainly broken. I've just fixed it in master.

**Reply by rafaelrenanpacheco** [reply]
`./gradlew publishToMavenLocal` is now working again, thanks for the help @alvarosanchez :+1:
